processing/src/main/java/org/apache/druid/segment/nested/NestedDataComplexTypeSerde.java

```diff
@@ -157,6 +160,34 @@ public byte[] toBytes(@Nullable Object val)
     };
   }
 
+  @Override
+  public <T extends Comparable<T>> TypeStrategy<T> getTypeStrategy()
+  {
+    return new ObjectStrategyComplexTypeStrategy<>(
+        getObjectStrategy(),
+        ColumnType.ofComplex(TYPE_NAME),
+        true,
+        new Hash.Strategy<Object>()
+        {
+          @Override
+          public int hashCode(Object o)
+          {
+            // TODO(laksh): VET, Check if StructuredData.wrap(o).hashCode() makes sense, given that most of the objects inside
```

Comment:
Is this todo still there ?
cc @clintropolis I think this is not a blocker for this PR. We can always improve the implementation of the StructuredData